### PR TITLE
Mvp

### PR DESCRIPTION
**Pull request by joshskkim**
@Team-Gandalf/dope Added a README and a puppeteer test

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -117,6 +117,19 @@ module.exports = {
       ],
       verbose: false,
     },
+    {
+      clearMocks: true,
```
Comment (sajjanrajvaidya) [inline jest.config.js:121]: Is this for puppeteer?

File: README.md [hunk 2]
```diff
@@ -1 +1,38 @@
-# Reviews
\ No newline at end of file
+# Reviews
```
Comment (sajjanrajvaidya) [inline README.md:1]: @sfshaider @BHamp Have you guys done this too?

**Review by sajjanrajvaidya — APPROVED** [review]
Dope

File: jest.config.js (hunk 1 above)
Comment (joshskkim) [reply jest.config.js:121]: Yes!

**Pull request merged**